server/routes/gyms.ts

```diff
@@ -12,6 +12,7 @@ gyms.post('/register', gymRegister);
 gyms.get('/top', getTopGyms);
 gyms.get('/filter', getFilteredGyms);
 gyms.get('/search', searchGymByName);
+gyms.post('/reviews', addReview);
```

Comment:
better to remove to `gym` router
```suggestion
gym.post('/review', addReview);
```